src/org/zaproxy/zap/extension/quickstart/QuickStartPanel.java

```diff
@@ -283,6 +299,20 @@ public void actionPerformed(java.awt.event.ActionEvent e) {
 		return confButton;
 	}
 
+	public void updateConfButton(boolean apiState) {
+		if (confButton == null) {
+			getConfButton();
+			return;
+		}
+		//PnH button has the same enable state as the API
+		confButton.setEnabled(apiState);
+		if (apiState) {
+			confButton.setToolTipText(Constant.messages.getString("quickstart.button.tooltip.mitm"));
+		} else {
+			confButton.setToolTipText(Constant.messages.getString("quickstart.mitm.api.disabled"));

```

Comment:
It should be used a ZapToggleButton which allows to specify the disabled tool tip text, so the tool tip texts do not need to be set manually.